wiring/inc/spark_wiring_cloud.h

```diff
@@ -75,6 +75,8 @@ class CloudClass {
 
 
 public:
+    static const size_t PUBLISH_VITALS_DISABLE;
```

Comment:
Why `size_t`?  Seems a little strange since the expected units is typically bytes,  but here we are expressing a duration. Is there a better type? In the HAL we define `system_tick_t` which might be a closer fit.

Reply:
This is a bizarre copy/paste error or something. It shouldn't be all caps and the data type was meant to be boolean. The variable below it is the `period` and it was type `bool` instead of `size_t`. Who knows? I was probably cross-eyed while typing.

Reply:
I see, so a mixup of the types. But is `size_t` the best type to use for duration? The units for `size_t` are bytes normally which doesn't seem to fit well with duration. On 64-bit platforms the duration will also be 64-bits which seems excessive! :-)

Reply:
`size_t` is the platform-native unsigned integer. It will be the most performant. Any other size type would bear the unnecessarily overhead of being masked when moving in and/or out of the device registers. From an efficiency perspective, as long as it is large enough, `size_t` seems like the right type to me.